src/main/java/org/jenkinsci/plugins/github/config/GitHubPluginConfig.java

```diff
@@ -171,7 +171,7 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
 
     @Override
     public String getDisplayName() {
-        return "GitHub Plugin Configuration";
+        return "GitHub Configuration";
```

Comment:
Yeah, "configuration" sounds bit redundant. Would it be possible to place some UI element that would show that this section comes from gh plugin, then i would be ok to remove "plugin" word?